projects/plugins/boost/app/assets/src/js/features/speed-score/pop-out/pop-out.module.scss

```diff
@@ -0,0 +1,39 @@
+.wrapper {
+	width: 400px;
+	position: absolute;
+	top: 40px;
+	right: 0;
+	z-index: 60;
+	overflow: hidden;
+	padding: 20px 0 20px 20px;
+	pointer-events: none;
+}
+
+.card {
+	box-shadow: 0 0 20px 0 rgb(170 170 170 / 28%);
+
+	transition: right 0.5s ease-in-out;
+
+	position: relative;
+	right: -100%;
+	padding: 24px;
+
+	background-color: white;
+	pointer-events: all;
+}
+
+/**
+ * Styles applied by CSSTransition when the component is fully visible.
+ */
+.appearDone, .enterDone {
+	right: 0%;
+}
+
+.dismissLink {
+	margin-top: 10px !important;
+	float: right;
+}
```

Comment:
Just noticed. We have a convention of using kebab case in CSS. I think it's probably ok to skip it for CSSTransition. However, I feel more comfortable with consistent case given that it's actually possible.

The `dismissLink` being not part of the CSSTransition, should be changed to kebab case.

Reply:
I've removed the camel case in c0e9305aaf, for the sake of this PR.

However, I would like to raise moving away from kebab-case in module scss files because array-style accessors (`style['dismiss-link']`) are harder to read.

Reply:
> However, I would like to raise moving away from kebab-case in module scss files because array-style accessors (style['dismiss-link']) are harder to read.

I just searched the Jetpack Monorepo, and there are by far more instances of `styles[ 'kebab-case' ]` than `styles.camelCase` so I'll bow to the popular convention. Consistency over personal preference.

Reply:
I like camelCase for the same convenience too. :(